setenv.sh

```diff
@@ -12,8 +12,10 @@ export KDBAPPCODE=${TORQAPPHOME}/code
 export KDBHDB=${TORQDATA}/hdb/database
 export KDBWDB=${TORQDATA}/wdbhdb
 
-export KDBBASEPORT=6000                                                                             # set KDBBASEPORT to the default value for a TorQ Installation
+export KDBBASEPORT=6000                                                                            # set KDBBASEPORT to the default value for a TorQ Installation
```

Comment:
looks like alignment changed on this comment bringing it out of line, please put it back

Reply:
will do